crates/mitex-web-wasm/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+//! This is a WASM wrapper of the MiTeX library for Web browsers.
+
+#![allow(missing_docs)]
+
+#[cfg(target_arch = "wasm32")]
+use wasm_bindgen::prelude::*;
+
+/// Converts the A LaTex math equation into a plain text
+///
+/// # Errors
+/// Returns an error if the input is not a valid utf-8 string
+/// Returns an error if the input is not a valid math equation
+#[cfg_attr(target_arch = "wasm32", wasm_bindgen)]
+pub fn convert_math(input: &str, spec: &[u8]) -> Result<String, String> {
```

Comment:
This is a same function as the one in mitex-typst. And we have to maintain comments on both. This is why I want to merge them.